independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4716 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "ORB Database",
+        "version": "1.0",
+        "description": "Orb provides with a framework of API calls to use the Orb database for various use cases. The framework comprises five calls, Match, Fetch, Search, Lookalike and Corporate Tree.",
+        "contact": {
+            "name": "Aaryan Arora, Ankita Singh",
+            "email": "aaryan2134@gmail.com"
+        }
+    },
+    "host": "api.orb-intelligence.com",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/3/match/": {
+            "get": {
+                "parameters": [
+                    {

```

Comment:
@aaryan2134  Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@sriyen-msft Could you please help me on this? I have changed and updated the indentation to 2 spaces. But still the file on github open in 8 spaces indentation view by default. I am really not able to understand why is this happening and how I can correct this. Could you please provide some help or guidance?

Reply:
I have tried using a plugin on VS code - Prettify Json, changing the indentation to 2 spaces. And it is working fine on VS code. When I commit on Github, it opens in 8 space view. I tried changing it by editing the file on Github itself. But it still resets to 8 spaces view.

Reply:
@aaryan2134 No other partner is seeing any issue with alignment in the swagger file while pushing commits to open source repo. Please update tab-index to 2 in your Visual Studio and validate your connector with paconn tool before pushing any commit to your PR.